src/utils/plot.py

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+from utils.matplotlib_util import Plot
+
+
+import argparse
+
+
+class PlotHelper:
```

Comment:
How do you run this? I tried running `python3 src/utils/matplotlib_util.py runs/cislunarsim-1650564840.722935.csv` but I don't see a matplotlib window open. Is this the right command? Please comment how to run this in the readme

Reply:
In the readme, it states that the command is
python3 src/utils/plot.py {file path}

Reply:
Oops forgot to pull all good 👍